src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,137 @@
+.cards {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 44px 46px;
+  margin: 50px 40px;
+
+  @media (max-width: 1023.98px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (max-width: 767.98px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 487.98px) {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+.product {
+  position: relative;
+  max-width: 200px;
+  cursor: pointer;
+  transition: color $transition, transform $transition;
+  box-sizing: border-box;
```

Comment:
Move it to the `*` selector
```suggestion
```

Reply:
![image](https://github.com/mate-academy/layout_catalog/assets/112823560/779391bf-b951-4b19-a0f8-ad41d0bfb92e)

we don't want to use